actionpack/lib/action_dispatch/middleware/static.rb

```diff
@@ -13,15 +13,13 @@ module ActionDispatch
   # located at `public/assets/application.js` if the file exists. If the file
   # does not exist, a 404 "File not Found" response will be returned.
   class FileHandler
-    def initialize(root, cache_control, index = 'index')
+    def initialize(root, index = 'index', headers: )
```

Comment:
Absolutely. See my comment above about it: https://github.com/rails/rails/pull/19135#discussion_r32110510